dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -1312,8 +1313,8 @@ public static void HydraTask_executeSnappyStreamingJobInTask() {
      */
     public static void HydraTask_executeSnappyJobInTask() {
```

Comment:
There should be only one method to execute jobs and the parmeters related to job should be provided at task level. otherwise it will be very difficult to relate global parmeters to particular task by the task name.